### ISSUE TITLE
kernel died on particular line

### ISSUE THREAD

**Issue by chandnikathuria1992** [jeffreyhuang1/two-stream-action-recognition#11]
Hi
On running code i am getting "kernel died and restarting" on line  self.bn1 = nn.BatchNorm3d( mid_planes).
if i comment this code line then it runs  but again same error occurs wherever batch normalization is used. Any idea what could be the reason.
